fix: links and more

weichweich opened the pull request:
## fixes no ticket

* Don't check links while building. We have a pipeline for that. It would be nice to have a preview even if links are broken.
* Fix broken link
* remove duplicate file
* fix extrinsic name in delegate guide

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

weichweich committed:
fix: links

weichweich committed:
fix: link

weichweich committed:
fix: wrong extrinsic name

ntn-x2 reviewed (commented):
I see that `check-links` is run by the CI, but what about `check-ts`? I personally like to call `build` locally before I push anything on the CI, so perhaps instead of changing that, we can add a new script `build:docs` which just generates the examples and calls docusaurus?

weichweich committed:
readd check-ts && check-links

weichweich commented:
> but what about `check-ts`?

Ah to be honest, I just thought was `check-links` twice when I removed that. Didn't notice that there was a `check-ts` instead. 😳 
I added both back. Maybe it's good that they are a bit annoying, so that nobody skips these checks. 😁

weichweich committed:
Update 01_vote.md
